### DISCUSSION TITLE
Get mouse-over info without the mouse

### DISCUSSION THREAD

**Discussion by ZachClayburn** [JetBrains/ideavim#283] [Q&A] [answered]
Say my IDE has found an issue in the file I am editing and I get a squiggly under some line. If I hover my mouse over that, I get more info on what is wrong. Is there a way to get that information on the text currently under my cursor without having to grab the mouse?

**Comment by AlexPl292** [comment, marked as answer]
Hey, there is no a separate IdeaVim action for that, but you can create a mapping to IJ action:
`map <c-x> <Action>(ShowErrorDescription)`.

**Reply by fleap-dev** [reply]
By hovering with the mouse, the IDE will show both documentation and error descriptions. Is it possible to make this keymap do both actions as well?

**Reply by AlexPl292** [reply]
Hi, there is no such shortcut at the moment: https://youtrack.jetbrains.com/issue/IDEA-249695/Show-error-messages-in-quick-doc-popup-invoked-by-keyboard-shortcut

**Reply by SkyLeite** [reply]
This is now possible with `map <c-x> <Action>(ShowHoverInfo)`

**Reply by ShaneMurphy2** [reply]
This doesn't seem to work anymore?

**Reply by emochka2007** [reply]
Not working in CLion, but works in webstorm...

**Reply by JustBarnt** [reply]
@ShaneMurphy2 @emochka2007 Seems like at least in Rider this action was renamed to `HoverInfo`. But is there away to show it and it not focus the modal window it shows the hover info in?

**Reply by ShaneMurphy2** [reply]
@JustBarnt Doesn't appear to be `HoverInfo` in CLion

**Reply by citizenmatt** [reply]
There aren't any known problems here, and it's working for me. It requires the IDE to be 2023.3 or later. The action is definitely called `ShowHoverInfo` and still available in the platform - it hasn't been renamed or removed or anything. If there are problems showing it, please can you share logs? It might be best to create an issue on youtrack.jetbrains.com in the VIM project, and attach logs there.

It's also not a modal window, but a tooltip, and you should be able to continue typing Vim commands while it's visible.
